docs/contributing/admins.md

```diff
@@ -15,6 +15,42 @@ This guide is for administrators of Plone Documentation.
 It covers automated deployments, hosting, automated testing, previewing, and importing external package documentation into Plone Documentation.
 
 
+(administrators-import-docs-submodule-label)=
+
+## Importing external docs and converting to MyST
+
+This section describes how to import external projects and convert their docs to MyST.
+We did this for `plone.app.dexterity` and several other projects.
+
+1.  Create a new branch using the name of the project.
+1.  Install [rst-to-myst](https://pypi.org/project/rst-to-myst/).
+
+    ```shell
+    bin/pip install "rst-to-myst[sphinx]"
+    ```
+
+1.  Clone the project repository to the root of `plone/documentation`.
+1.  Delete any non-documentation files from the clone.
+1.  Move the documentation files and subfolders to the root of the clone, retaining the documentation structure.
+1.  Convert the reStructuredText documentation files to MyST.
+    The example commands below assume that there are files at the root of the clone and in one sub-level of nested directories. For deeper nesting, insert globbing syntax for each sub-level as `**/`
```

Comment:
```suggestion
    The example commands below assume that there are files at the root of the clone and in one sub-level of nested directories.
    For deeper nesting, insert globbing syntax for each sub-level as `**/`
```

Reply:
Good catch. Anything else? OK to merge?

Reply:
Whoops. nm. Too early. I still have a lot more work ahead with cleaning up the converted MyST files.